core/ulog/log.go

```diff
@@ -60,21 +60,45 @@ type baselogger struct {
 type Log interface {
 	// Configure Log object with the provided log.Configuration
 	Configure(Configuration)
-	// Create a child logger with some context
-	With(...interface{}) Log
-	SetLevel(zap.Level)
-	SetLogger(zap.Logger)
-	Check(zap.Level, string) *zap.CheckedMessage
+
+	// With creates a child logger with the provided paramets as key value pairs
```

Comment:
s/paramets/parameters/

> with the provided paramets as key value pairs

what does that mean to the user? Is it equivalent to passing these key-value pairs to all subsequent logging calls on that logger?